packages/storefront-lwc/src/modules/commerce/productSearchResults/productSearchResults.js

```diff
@@ -5,38 +5,106 @@
     For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 */
 import { LightningElement, wire, track, api } from 'lwc';
-import { productsByQuery } from 'commerce/data';
+// import { productsByQuery } from 'commerce/data';
 
-//
+import { useQuery } from '@lwce/apollo-client';
+import gql from 'graphql-tag';
+import { apiClient } from '../api/client';
+
+// const QUERY = gql``;
+const QUERY = gql`
+    query($query: String!, $filters: [Filter]) {
+        productSearch(query: $query, filterParams: $filters) {
+            productHits {
+                productId
+                productName
+                prices {
+                    sale
+                    list
+                }
+                image {
+                    title
+                    link
+                    alt
+                }
+                colorSwatches {
+                    name
+                    value
+                    title
+                    link
+                    alt
+                    style
+                }
+            }
+            refinements {
+                values {
+                    label
+                    value
+                    hitCount
+                    values {
+                        label
+                        value
+                        hitCount
+                    }
+                }
+                label
+                attributeId
+            }
+            currentFilters {
+                id
+                value
+            }
+        }
+    }
+`;
 // Displays search results
 //
 export default class ProductSearchResults extends LightningElement {
     @track state = {};
     @track products = [];
     @track refinementgroups = [];
-    @api query = '';
+    // @api query = '';
     @track loading = false;
     @track refinementBar = 'refinement-bar col-md-3 d-none d-lg-block';
     @track showRefinementBar = true;
     sortRule;
     selectedRefinements = {};
 
+    variables = {
+        query: '',
+        sortRule: '',
+        filters: [],
+    };
+
+    @api set query(val) {
+        this.variables = { ...this.variables, query: val };
+    }
+
+    get query() {
+        return this.variables.query;
+    }
+
     // The wire adaptor to search for products when the query, sort rule or selected refinements change.
-    @wire(productsByQuery, {
-        query: '$query',
-        sortRule: '$sortRule',
-        selectedRefinements: '$selectedRefinements',
+    // @wire(productsByQuery, {
+    //     query: '$query',
+    //     sortRule: '$sortRule',
+    //     selectedRefinements: '$selectedRefinements',
+    // })
```

Comment:
remove these commented code if you don't need it.

Reply:
done, pr has moved to https://github.com/SalesforceCommerceCloud/sfcc-sample-apps/pull/162